src/vivarium_nih_us_cvd/model_specifications/branches/scenarios.yaml

```diff
@@ -2,6 +2,18 @@ input_draw_count: 10
 random_seed_count: 15
 
 branches:
-# TODO change to name of intervention
-  - placeholder_branch_name:
-      scenario: ['baseline']
+  - intervention:
+      scenario: "baseline"
+      outreach_scale_up:
```

Comment:
Per our conversation, the `outreach_scale_up` blocks should be unindented one level